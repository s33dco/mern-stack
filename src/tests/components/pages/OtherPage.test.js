import React from 'react';
import { shallow } from 'enzyme';
import OtherPage from '../../../components/pages/OtherPage';

let wrapper;

beforeEach(() => {
	wrapper = shallow(<OtherPage />);
});

describe('OtherPage component', () => {
	it('should render OtherPage component correctly', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
