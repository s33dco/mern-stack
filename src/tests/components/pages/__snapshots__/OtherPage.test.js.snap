// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OtherPage component should render OtherPage component correctly 1`] = `ShallowWrapper {}`;
